baseplate.go

```diff
@@ -217,22 +217,7 @@ func Serve(ctx context.Context, args ServeArgs) error {
 	return <-shutdownChannel
 }
 
-// parseConfig parses from the YAML file at the given path into cfg.
-func parseConfig(path string, cfg Configer) error {
-	if path == "" {
-		return errors.New("baseplate.ParseConfig: no config path given")
-	}
-
-	f, err := os.Open(path)
-	if err != nil {
-		return err
-	}
-	defer f.Close()
-
-	return DecodeConfigYAML(f, cfg)
-}
-
-// DecodeConfigYAML decods the YAML read from the given Reader into cfg.
+// ParseConfigYAML parses path as YAML into cfg.
```

Comment:
Now where this is called explicitly, do we need to call out the fact that this is supposed to be called before passing it to `baseplate.New` or do we not necessarily care as it should be given through the baseplate-cookiecutter? Also trying to think if any steps are necessary to prevent problems during upgrading if someone misses the release notes about the change.

Reply:
yes there will be a cookiecutter update after the release, and this will also be noted in the release note.

people can't blindly upgrade, `baseplate.NewArgs.ConfigPath` is removed so their code won't compile :)